### PR TITLE
[SE-0298: Async/Await: Sequences] Enable by default.

### PR DESCRIPTION
**Pull request by DougGregor**
One still needs to import the _Concurrency library to use it.

### CODE REVIEW

**Comment by DougGregor** [comment]
@swift-ci please smoke test

**Comment by DougGregor** [comment]
@swift-ci please test source compatibility

File: CHANGELOG.md [hunk 1]
```diff
@@ -8309,6 +8322,7 @@ Swift 1.0
 [SE-0286]: <https://github.com/apple/swift-evolution/blob/main/proposals/0286-forward-scan-trailing-closures.md>
 [SE-0287]: <https://github.com/apple/swift-evolution/blob/main/proposals/0287-implicit-member-chains.md>
 [SE-0296]: <https://github.com/apple/swift-evolution/blob/main/proposals/0296-async-await.md>
+[SE-0298: <https://github.com/apple/swift-evolution/blob/main/proposals/0298-asyncsequence.md>
```
Comment (slavapestov) [inline CHANGELOG.md:8325]: Syntax error here, missing ]

Comment (DougGregor) [reply CHANGELOG.md:8325]: Bah, thanks!

**Comment by DougGregor** [comment]
@swift-ci please smoke test

**Comment by DougGregor** [comment]
@swift-ci please test source compatibility

**Pull request merged**